.env.example

```
@@ -168,6 +168,13 @@ MEMHOUSE_RETRIEVAL_RERANK_RESERVED_MS=120
 MEMHOUSE_RETRIEVAL_ENTITY_FREQUENCY_CEILING=0.5
 MEMHOUSE_RETRIEVAL_ENTITY_CEILING_MIN_STATEMENTS=20
 MEMHOUSE_RETRIEVAL_ENTITY_PER_ENTITY_CAP=25
+# Expansion uses only the interleaved seed head. Each seed can add at most this
+# many shared-entity neighbours. Ubiquitous entities are ignored once the
+# visible corpus reaches the configured minimum size.
+MEMHOUSE_RETRIEVAL_EXPAND_SEED_LIMIT=10
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MEMHOUSE_RETRIEVAL_EXPAND_SEED_LIMIT key should go before the MEMHOUSE_RETRIEVAL_RERANK_RESERVED_MS key

(UnorderedKey)